source/localizable/tutorial/acceptance-test.md

```diff
@@ -0,0 +1,145 @@
+The purpose of ember acceptance tests are to provide UI automated tests from a user perspective.
+They interact with the rendered webpage as an actual user of the system would.
+Acceptance tests are well suited for testing high level goals of your application, such as the ones listed above for Super Rentals.
+
+We'll start by using Ember CLI to generate a new acceptance test:
+
+```shell
+ember g acceptance-test list-rentals
+```
+
+The command will generate the following output, showing that it created a single file called `list-rentals-test`.
+
+```shell
+installing acceptance-test
+  create tests/acceptance/list-rentals-test.js
+```
+
+Opening the new test file will reveal some boilerplate code that will try to go to the `list-rentals` route and verify that the route is loaded.
+This boiler plat code is there to guide you into your first working acceptance test.
+Since we are testing our index route, which is `/`, we'll replace occurances of `/list-rentals` with `/`.
+
+```javascript{-6,+7,-8,+9,-12,+13}
+import { test } from 'qunit';
+import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | list-rentals');
+
+test('visiting /list-rentals', function(assert) {
+test('visiting /', function(assert) {
+  visit('/list-rentals');
+  visit('/');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/list-rentals');
+    assert.equal(currentURL(), '/');
+  });
+});
+```
+
+Now run your test suite with `ember t -s` from the command line and you'll see one successful test!
+
+As mentioned before, this test boilerplate is just for checking the environment, so now lets replace this test with our list of goals.
+
+```/tests/acceptance/list-rentals-test.js
+import { test } from 'qunit';
+import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | homepage');
+
+test('should list available rentals.', function (assert) {
+});
+
+test('should display some key information about each rental in the list.', function (assert) {
+});
+
+test('should link to information about the company.', function (assert) {
+});
+
+test('should link to contact information.', function (assert) {
+});
+
+test('should filter the list of rentals by city.', function (assert) {
+});
+```
+
+These tests will fail, since ember tests will always fail without assertions.
+Since we have an idea of what we want our application to look like, we can also add some details to the tests.
+
+Ember acceptance tests provide test helpers to perform common tasks, such as visiting routes, filling in fields, clicking on elements, and waiting on renders.
+
+To check that rentals are listed, we'll first visit the index route and check that the results contain 3 listings.
+
+```/tests/acceptance/list-rentals-test.js
+test('should list available rentals.', function (assert) {
+  visit('/');
+  andThen(function () {
+    assert.equal(this.$('.results .listing').length, 3);
```

Comment:
I tend to add in the test text to make this easier to debug if things break in the future.  Plus it helps document what I'm expecting.  Could we do that here?  It would help explain what we're expecting and might eliminate sentences in the text ...

`assert.equal(this.$('.results .listing').length, 3, "We should see 3 listings");